### PR TITLE
Missing translations

### PR DESCRIPTION
**Pull request by nim-odoo**
When a view cannot be loaded (e.g. when html tag errors in translation),
we log the the xmlid of the view and the language in which it happened.

### CODE REVIEW

**Comment by xmo-odoo** [comment]
* how comes this exception isn't logged when it bubbles up somewhere else? If it is and the goal is just to provide some more information about loading failure (key and lang) why not log *just that* with `logger.error`?
* why not re-raise the original exception as-is and keep all the information it contains?
* `logger.exception` will log the traceback and exception's error message, no need to re-log `e`, that just duplicates the exception's description/message
* don't manually format logging messages, let logging itself do that

**Comment by nim-odoo** [comment]
@xmo-odoo : it is replaced by https://github.com/odoo/odoo/pull/10347 anyway, it won't be merged 

But thanks!

File: addons/hr_equipment/i18n/hr_equipment.pot [hunk 1]
```diff
@@ -105,7 +105,7 @@ msgstr ""
 
 #. module: hr_equipment
 #: model:ir.model,name:hr_equipment.model_hr_equipment_category
-#: model:ir.model.fields,field_description:hr_equipment.field_hr_equipment_category_id_5383
+#: model:ir.model.fields,field_description:hr_equipment.field_hr_equipment_category_id_6493
```
Comment (mart-e) [inline addons/hr_equipment/i18n/hr_equipment.pot]: this looks suspicious

File: addons/web/static/src/js/action_manager.js [hunk 2]
```diff
@@ -19,7 +21,7 @@ var Widget = require('web.Widget');
 var Action = core.Class.extend({
     init: function(action) {
         this.action_descr = action;
-        this.title = action.display_name || action.name;
+        this.title = _t(action.display_name || action.name);
```
Comment (mart-e) [inline addons/web/static/src/js/action_manager.js]: Isn't that too late to be translated? What about the export?

Comment (nim-odoo) [inline addons/web/static/src/js/action_manager.js]: Yes, replaced by https://github.com/odoo/odoo/commit/1c4d888e80f5e28de1c682e6823dde59d89965e3

**Pull request merged**